sql/binding_contexts.cc

```diff
@@ -25,7 +25,7 @@ BoundSqlExpression
 ReadThroughBindingContext::
 rebind(BoundSqlExpression expr)
 {
-    auto outerExec = std::move(expr.exec);
```

Comment:
This was trying to implement a kind of swap.  Not sure why it did not work.  May be a too aggressive compiler optimization?  This could explain why it was working in debug.